CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 * [\#73](https://github.com/Finschia/wasmd/pull/73) test: add the check for expPaginationTotal
 * [\#72](https://github.com/Finschia/wasmd/pull/72) add pagination next key test in ContractHistory
 * [\#75](https://github.com/Finschia/wasmd/pull/75) test: add the test case for InactiveContract
+* [\#69](https://github.com/Finschia/wasmd/pull/69) refactor: add test cases to Params
```

Comment:
Do we need a prefix here?
(Some of them are already merged...)

Reply:
Thanks. The change.log will be refactored together at the end. Then remove the prefix.